src/App.js

```diff
@@ -67,26 +218,71 @@ function App() {
             title: itemName,
             completed: false,
             priority: priority, // planning on tiny, medium, high
-            created: firebase.database.ServerValue.TIMESTAMP
+            created: firebase.database.ServerValue.TIMESTAMP,
+            owner: props.user.uid,
```

Comment:
Why have owner and sharedWith fields in your tasks?  If you were going to have it, wouldn't the sharedWith need to be initialized to the sharedWith of its list?